ui/src/app/cohort-search/cohort-criteria-menu.spec.tsx

```diff
@@ -33,7 +33,6 @@ describe('CohortCriteriaMenu', () => {
     serverConfigStore.set({
       config: {
         ...defaultServerConfig,
-        enableDrugWildcardSearch: false,
       },
     });
```

Comment:
Since the component no longer references `serverConfigStore` we can remove it from the test also.